rune/libenclave/internal/runtime/pal/skeleton/sgxsign.c

```diff
@@ -255,7 +255,7 @@ static bool measure_encl(const char *path, uint8_t *mrenclave,
 	if (!ctx)
 		return false;
 
-	file = fopen(path, "rb");
+	file = fopen(path, "r+b");
```

Comment:
What is the problem of "rb"?

Reply:
`rb`: Open a binary file for reading.
`r+b`: Open a binary file for both reading and **writing.**
I need write the value of `max-mmap-size` to the metadata area in enclave.bin. So I set `r+b` File Mode